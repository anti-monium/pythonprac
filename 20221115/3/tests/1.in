try:
	alp = Alpha(abc=903)
except AttributeError:
	print('Well done')	
alp = Alpha(a=9, b=0, c=3)
try:
	alp.stp = 123
except AttributeError:
	print('Well done')
alp.s, alp.t, alp.p = 1, 2, 3
print(alp)
try:
	alp = AlphaQ(abc=903)
except AttributeError:
	print('Well done')	
alp = AlphaQ(a=9, b=0, c=3)
try:
	alp.stp = 123
except AttributeError:
	print('Well done')
alp.s, alp.t, alp.p = 1, 2, 3
print(alp)
