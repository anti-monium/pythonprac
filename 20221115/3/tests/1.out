Well done
Well done
a: 9, b: 0, c: 3, p: 3, s: 1, t: 2
Well done
Well done
a: 9, b: 0, c: 3, p: 3, s: 1, t: 2
